docs/src/appendix/snforge-library/fs.md

```diff
@@ -2,28 +2,6 @@
 
 Module containing functions for interacting with the filesystem.
 
-## `File`
-```
-trait FileTrait {
-    fn new(path: ByteArray) -> File;
-}
-```
-
-## `FileParser<T>`
-```rust
-trait FileParser<T, +Serde<T>> {
-    fn parse_txt(file: @File) -> Option<T>;
-    fn parse_json(file: @File) -> Option<T>;
-}
-```
-
-## `read_txt` & `read_json`
-```rust
-fn read_txt(file: @File) -> Array<felt252>;
-fn read_json(file: @File) -> Array<felt252>;
-```
-
-
 ## File format
```

Comment:
This section could be improved by changing the formatting and wording. Some suggestion below since it's not letting me comment on specific lines

Reply:
Let's use cairo terms for these - say that plain text files can contain valid Felts or Felts252, don't think we need to be explaining what they represent here.

Reply:
Lets just say shortstrings and strings can be used (without calling them single line short string), you can say that they cannot be longer than single line but let's reword that.

Reply:
I'd have it structured as something similar to this

- plain text
  * example
  * rules
- json
   * example
   * rules

And inline the comments with the examples